lib/tasks/govuk_assets.rake

```diff
@@ -16,4 +16,15 @@ namespace :govuk_assets do
     status = result.successful? ? 'OK' : 'Error'
     puts "#{status}: #{result.written_count} documents updated"
   end
+
+  desc 'Create a whitehall asset with the given legacy URL path'
+  task :create_whitehall_asset, [:file_path, :legacy_url_path] => :environment do |_, args|
+    file_path = args[:file_path]
+    legacy_url_path = args[:legacy_url_path]
+    asset = WhitehallAsset.new(
```

Comment:
To save lines, you can actually do a `WhitehallAsset.create!(` instead of a `new` and then a `save`.